core.yml

```diff
@@ -740,6 +726,29 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/BadRequestError'
+              examples:
+                    Bad Request:
+                     value:
+                      type: list
+                      correlationId: ac5ah5i
+                      data:
+                        - type: error
+                          title: Parameter not valid.
+                          code: parameter-not-valid
+                          detail: ID value is not valid.
+                          source:
+                            parameter: id
+                    Invalid Filters:
+                      value:
+                        type: list
+                        correlationId: ac5ah55
+                        data:
+                          - type: error
+                            code: parameter-not-valid
+                            title: Parameter value is not valid
+                            detail: The provided filter parameter is in invalid format or unsupported
+                            source:
+                              parameter: filter
         '403':
```

Comment:
Noticed a trailing comma on the end of this 

`Error that access is forbidden and tied to the application logic,`

is that correct?

Reply:
Yep, that's correct.